src/System.IO.Abstractions.TestingHelpers/MockFileData.cs

```diff
@@ -126,17 +126,32 @@ public string TextContents
         /// <summary>
         /// Gets or sets the date and time the <see cref="MockFileData"/> was created.
         /// </summary>
-        public DateTimeOffset CreationTime { get; set; } = new DateTimeOffset(2010, 01, 02, 00, 00, 00, TimeSpan.FromHours(4));
```

Comment:
Isn't this a user visible behavior change, now that for this and all the other xxxTime Properties below are no longer initialized?

Reply:
You are right, I should probably initialize them in the the constructor.
But are the "Dummy" times really the desired behavior or should I not rather initialize them to either `DateTime.UtcNow` or `DefaultDateTimeOffset`?

Reply:
This is only relevant when directly creating the `MockFileData` (when using the methods from the `IFileSystem`, the time is adjusted correctly). Therefore I suggest to change the constructor to initialize them with `DateTime.UtcNow` as this is the same behaviour of the real file system, even if this is a user visible behaviour change (relying on explicit dummy-values seems not a good approach for me in any case):
```csharp
        /// <summary>
        /// Initializes a new instance of the <see cref="MockFileData"/> class with an empty content.
        /// </summary>
        private MockFileData()
        {
            var now = DateTime.UtcNow;
            LastWriteTime = now;
            LastAccessTime = now;
            CreationTime = now;
        }
```

Reply:
Looks reasonable, but as stated elsewhere, the other ctors also need to be handled